### ISSUE TITLE
Problems building pdf with nested tables

### ISSUE THREAD

**Issue by andygotz** [tango-controls/tango-doc#113]
The Hdb++  documentation has nested tables in the device interface documentation. When trying to build the pdf version on my laptop using the command "make latexpdf" I get this error:

> Markup is unsupported in LaTeX:
> administration/services/hdbpp/hdbpp-es_doc/FullDocument-hdbpp-es:: nested tables are not yet implemented.
> Makefile:114: recipe for target 'latexpdf' failed
> make: *** [latexpdf] Error 1
> 

I see the nested tables are rendered on the html version but how to get them rendered in pdf? The pdf version on readthedocs does not seem to have them either.

**Comment by piogor** [comment]
Yes, the RtD builder is reporting the same issue: http://readthedocs.org/projects/tango-controls/builds/5830248/

**Comment by andygotz** [comment]
I wonder if we need these tables in the documentation? Maybe we can remove them for now.

**Comment by piogor** [comment]
I have already started splitting, but I may remove them instead.

**Comment by andygotz** [comment]
I would remove this file for now. It is very detailed and should be part of the auto generated doxygen documentation.

**Comment by piogor** [comment]
There is also https://github.com/tango-controls/tango-doc/blob/master/source/administration/services/hdbpp/hdbpp-cm_doc/FullDocument-hdbpp-cm.rst  file which is generated documentation. There, the tables are un-nested. Shell I remove it, too?

**Comment by andygotz** [comment]
Yes
